README.md

```diff
@@ -13,9 +13,14 @@ $ cp .env.sample .env
 Use the script to create your zip archive:
 
 ```
-$ npm run zip
+$ grunt run
```

Comment:
Can this be added as an alternative to the zip script + manual uploading to AWS? I'd like to document both options for folks that would rather upload their zip some other way.